docs/reference-guides/core-blocks.md

```diff
@@ -20,7 +20,7 @@ Displays a group of accordion headings and associated expandable content. ([Sour
 
 ## Accordion Heading
 
-Displays an accordion heading. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion-heading))
+Displays a clickable heading that toggles the content panel open and closed. ([Source](https://github.com/WordPress/gutenberg/tree/trunk/packages/block-library/src/accordion-heading))
```

Comment:
I wonder if there's a term other than "clickable", especially since the "toggles" term shows up later. Perhaps something simple:

> Toggles the accordion panel.

A bit bare, perhaps, but simple and maybe reads well.